Peer/Program.cs

```diff
@@ -41,17 +42,11 @@ private static AppConfig BuildConfig()
             return configModel;
         }
 
-        private static void BuildServices(IServiceCollection services, out PeerPullRequestAPI<GithubPullRequestSource> GitpRListApi, out PeerPullRequestAPI<AzdoPullRequestSource> AzDopRListApi, out PeerPullRequestAPI<GitlabPullRequestSource> GitlabpRListApi)
+        private static void BuildServices(ServiceProvider serviceProvider, out PeerPullRequestAPI<GithubPullRequestSource> GitpRListApi, out PeerPullRequestAPI<AzdoPullRequestSource> AzDopRListApi, out PeerPullRequestAPI<GitlabPullRequestSource> GitlabpRListApi)
         {
-            GitpRListApi = services
-                            .BuildServiceProvider()
-                            .GetRequiredService<PeerPullRequestAPI<GithubPullRequestSource>>();
-            AzDopRListApi = services
-                            .BuildServiceProvider()
-                            .GetRequiredService<PeerPullRequestAPI<AzdoPullRequestSource>>();
-            GitlabpRListApi = services
-                            .BuildServiceProvider()
-                            .GetRequiredService<PeerPullRequestAPI<GitlabPullRequestSource>>();
+            GitpRListApi = serviceProvider.GetRequiredService<PeerPullRequestAPI<GithubPullRequestSource>>();
+            AzDopRListApi = serviceProvider.GetRequiredService<PeerPullRequestAPI<AzdoPullRequestSource>>();
+            GitlabpRListApi = serviceProvider.GetRequiredService<PeerPullRequestAPI<GitlabPullRequestSource>>();
```

Comment:
gitLabPRListApi
pr
gitLabPrListApi 

pR doesn't fit with any scheme I know of